docs/openapi/components/schemas/common/FSMAReceivingCTE.yml

```diff
@@ -0,0 +1,194 @@
+$linkedData:
+  term: FSMAReceivingCTE
+  '@id': https://w3id.org/traceability#FSMAReceivingCTE
+title: FSMA Receiving CTE
+description: >-
+  "Information regarding an FSMA Receiving CTE (Critical Tracking Event), designed to meet
+  reporting requirements for the FDA Traceability Rule as specified here:
+  https://www.fda.gov/food/food-safety-modernization-act-fsma/fsma-proposed-rule-food-traceability"
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FSMAReceivingCTE
+      - type: string
+        const: FSMAReceivingCTE
+  shipment:
+    title: Shipment
+    description: The shipment being tracked.
+    $ref: ./FSMAShipment.yml
+    $linkedData:
+      term: shipment
+      '@id': https://w3id.org/traceability#FSMAShipment
+  dateReceived:
+    title: Date Received
+    description: The date shipment was received in 24hr ISO 8601 format (e.g. 1970-01-01T00:00Z for UTC, or 1970:01:01T00:00-5:00 for New York on standard time).
+    type: string
+    $linkedData:
+      term: dateReceived
+      '@id': https://schema.org/DateTime
+  additionalData:
+    title: Additional Data
+    description: Additional Key Data Elements (KDEs).
+    type: array
+    items:
+      $ref: ./FSMAAbstractKDE.yml
+    $linkedData:
+      term: additionalData
+      '@id': https://w3id.org/traceability#FSMAAbstractKDE
+additionalProperties: false
+example: |-
+  {
+    "type": "FSMAReceivingCTE",
+    "shipment": {
+      "type": "FSMAShipment",
+      "product": {
+        "type": "FSMAProduct",
+        "traceabilityLot": {
+          "type": "FSMATraceabilityLot",
+          "lotCode": "CHE-MIG-TTF1-061321-H37J",
+          "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date and a random 4 digit string.",
+          "lotCodeGeneratorLocation": {
+            "type": [
+              "Place"
+            ],
+            "address": {
+              "type": [
+                "PostalAddress"
+              ],
+              "organizationName": "John's Produce",
+              "streetAddress": "4335 Walsh Underpass",
+              "addressLocality": "Port Mark",
+              "addressRegion": "LA",
+              "postalCode": "67391",
+              "addressCountry": "United States"
+            },
+            "locationName": "John's Tomato Farm #1"
+          },
+          "lotCodeGeneratorPOC": {
+            "type": "Entity",
+            "entityType" : "Person",
```

Comment:
You will have lots of undefined terms

Reply:
@OR13 This is another case where I am trying to reference our Entity schema. Is there a better way to do this?